frontend/app/src/people/utils/language_label_style.ts

```diff
@@ -290,3 +290,92 @@ export const colourOptions: readonly ColourOption[] = [
     color: '#FF8D85'
   }
 ];
+
+export const BountyDetailsCreationData = {
```

Comment:
let try creating a new file in `./src/form/formConstants.ts` it doesn't make sense to have this data in the `language_label_style.ts`

Reply:
oh wait I see why you put that there, form is in a whole separate folder than people

Reply:
maybe put it in another file named `bountyCreationConstants.ts` in `frontend/app/src/people/utils`